downstream/modules/platform/ref-eda-system-requirements.adoc

```diff
@@ -2,7 +2,13 @@
 
 = {EDAcontroller} system requirements
 
-The {EDAcontroller} is a single-node system capable of handling a variable number of long-running processes (such as, Rulebook activations) on-demand, depending on the number of CPU cores. Use the following minimum requirements to execute a maximum of 9 simultaneous activations:
+The {EDAcontroller} is a single-node system capable of handling a variable number of long-running processes (such as, Rulebook activations) on-demand, depending on the number of CPU cores. Use the following minimum requirements to execute, by default, a maximum of 12 simultaneous activations:
```

Comment:
Punctuation suggestion: remove the comma splice after "such as" -- no comma is needed there.

Reply:
good point. Pretty simple update to handle.